### PR TITLE
Added HTML sub-tab to FD2 School tab

### PR DESCRIPTION
**Pull request by andrewscheiner53**
__Pull Request Description__

Addresses #4  

---
__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

**Comment by andrewscheiner53** [comment]
For the commit `e05d63c ` I already added the "hr" (horizontal line) and "br" (break in text) tags to my sample Harvest Report page. The next commit will just include "mockup" being italicized.

File: farmdata2/farmdata2_modules/fd2_school/HTML/html.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<head>
+    <body>
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_school/HTML/html.html:3]: The <body> tag should not be within the <head> tag.  In a sub-tab we do not actually need either the <head> tag or the <body> tag.  But if you have them they should not be nested.

File: farmdata2/farmdata2_modules/fd2_school/HTML/html.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <h1>Harvest Report</h1>
+        <p>This page is a <i>mockup</i> of a simplified harvest report.</p>
+        <p>
+            <label for="title">Report Title:</label>
+            <input id="title" value="Sample title"/>
```
Comment (braughtg) [inline farmdata2/farmdata2_modules/fd2_school/HTML/html.html:8]: This input element should have a "type" attribute to make it a text field

**Comment by braughtg** [comment]
FD2School 01 - HTML Spike Feedback:

* Overall
  - [x] - added HTML sub-tab to the FD2 School Tab (4, 5, 6)
  - [x] - Code in `.module` file is neat
  - [ ] - Made PR with descriptive body. (8)
  - [x] - PR Body contains Link to the issue. (8)
  - [x] - Commits for nameable units of work. 
  - [x] - Uses descriptive commit messages.
  - [x] - PR Converted from draft to regular (30)

* HTML Sub-tab contents:
  - [x] - HTML code is neatly formatted.
  - [x] - Headers (14, 15)
  - [x] - All Details (farm, user, language, start, end, crop) (14, 15)
  - [x] - Italics, horizontal rules and line breaks (17, 18)
  - [x] - Form Elements (crop, area, date) (25, 26)
  - [x] - Table (headers, 2+ data rows) (28-29)

* Comments:
  * Assignment is complete so this PR has been closed.
  * Your score will be posed in Moodle.

**Pull request closed without merging**